docs/bis/guide/fundamentals/federationGuids.md

```diff
@@ -0,0 +1,25 @@
+# FederationGuids
+
+Every BIS `Element` has an optional 128 bit [Globally Unique Identifier](https://en.wikipedia.org/wiki/Universally_unique_identifier) called `FederationGuid`. It is really identifying the real-world Entity that the Element represents, not the Element itself. Generally it is intended that FederationGuids are assigned by external systems to *federate* Elements to their external meaning.
+
+## Scope of Uniqueness
+
+Assigned FederationGuids are enforced to be unique on a single BIS Repository. However, multiple BIS Repositories under the same iTwin can use the same FederationGuid in order to refer to the same real-world Entity.
```

Comment:
do we need 'under the same iTwin' here ... if by iTwin we mean context then it is feasible that the same entity may exist in iModels spanning multiple contexts.  I think we can just drop this bit of the sentence because scoping it within one iTwin does not add much over saying 'multiple BIS repositories'